client/tests/integration/offline_peers.rs

```diff
@@ -43,3 +40,41 @@ fn genesis_block_is_committed_with_some_offline_peers() -> Result<()> {
     assert_eq!(AssetValue::Quantity(alice_has_roses), *asset.value());
     Ok(())
 }
+
+#[test]
+fn register_offline_peer() -> Result<()> {
+    let n_peers = 4;
+
+    let (_rt, network, client) = Network::start_test_with_runtime(n_peers, Some(11_160));
+    wait_for_genesis_committed(&network.clients(), 0);
+    let pipeline_time = Configuration::pipeline_time();
+    let peer_clients = Network::clients(&network);
+
+    check_status(&peer_clients, 1);
+
+    let address = "128.0.0.2:8085".parse()?;
+    let key_pair = KeyPair::generate().unwrap();
+    let public_key = key_pair.public_key().clone();
+    let peer_id = PeerId::new(&address, &public_key);
+    let register_peer = Register::peer(DataModelPeer::new(peer_id));
+
+    // Wait for some time to allow peers to connect
+    client.submit_blocking(register_peer)?;
+    std::thread::sleep(pipeline_time * 2);
+
+    // Make sure status hasn't change
+    check_status(&peer_clients, 2);
```

Comment:
But status has changed from `1` to `2`, no?

Reply:
that's the block height. One block was produced

Reply:
the comment is misleading